types/Errors.d.ts

```diff
@@ -1,5 +1,8 @@
 declare module 'stripe' {
   namespace Stripe {
+    /**
+     * @deprecated The {@link Stripe.Errors} type is deprecated, please use {@link Stripe.errors} namespace directly.}
```

Comment:
This deprecation only matters for customers that for some reason used Stripe.Errors type directly, which I expect to be very rare.